docs/cases/indicators-of-compromise.asciidoc

```diff
@@ -92,3 +92,37 @@ A new Timeline opens with an automatically populated KQL query. The query contai
 `file.hash.sha256 : c207213257a63589b1e1bd2f459b47becd000c1af8ea7983dd9541aff145c3ba`.
 
 Using a KQL query that includes both the indicator and source event allows Timeline to find all events and alerts that have matching field-value pairs.
+
+[discrete]
+[[attach-indicator-to-case]]
+== Attach indicators to cases
+
+Provide more context to your investigations by attaching indicators to cases:
+
+. From the Indicators table and click the *More actions menu* *(…​)*. Alternatively, open an indicator's details, then select *Take action*.
+. Select one of the following:
+.. *Add to existing case*: From the *Select case* dialog box, select the case to which you want to attach the indicator. 
+.. *Add to new case*: Configure the case details. Refer to <<cases-ui-open>> to learn more about opening a new case. 
+
+The indicator is added to the case as a new comment. 
+
+[role="screenshot"]
+image::images/indicator-added-to-case.png[Shows an indicator attached to a case]
+
+[discrete]
+[[review-indicator-in-case]]
+=== Review indicator details in cases
+
+When you attach an indicator to a case, the indicator is added a new comment with the following details:
+
+* *Indicator name*: The name of the indicator. Clicking on the linked name opens the Indicator details flyout, which contains additional information about the indicator. Indicator details are provided in JSON format.
+* *Feed name*: The name of the threat feed from which the indicator was ingested.
```

Comment:
Maybe omit some of the more repetitive details, make the actions more straightforward:
```suggestion
* *Indicator name*: Click the linked name to open the Indicator details flyout, which contains additional information about the indicator. Indicator details are in JSON format.
* *Feed name*: The threat feed from which the indicator was ingested.
```